src/modules/af-liste/innslag/ListeInnslag.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { ListeTittel } from "../tittel/ListeTittel";
+import { CheckAndPrint } from "../../../components/check-and-print/CheckAndPrint";
+import { CheckPeriodAndPrint } from "../../../components/check-period-and-print/CheckPeriodAndPrint";
+import sprak from "../../../language/provider";
```

Comment:
Burde sprak også være named export hvis vi kjører det som praksis rundt om?

Reply:
Tenkte å ta det i neste runde for å ikke pirke innom alt for mange filer i denne PRen 😅  Har kun endret i components- og modules-mappene i denne omgangen.